### ISSUE TITLE
Crash server 1.12.2

### ISSUE THREAD

**Issue by TheTestMod** [Magneticraft-Team/Magneticraft#145]
java.lang.NoClassDefFoundError: com/cout970/modelloader/internal/CustomModelLoader

Full crashlog: https://pastebin.com/bF8hGcK2

**Comment by cout970** [comment]
Use ModelLoader 1.1.3, 1.1.4 has a bug and doen't work on servers. both versions work perfectly with magneticraft

**Comment by TheTestMod** [comment]
Thanks!

**Comment by TheSnowyChickens** [comment]
One moment:
MC (Client) need the ML 1.1.5 (so that MC can start up), BUT the server only work with ML 1.1.3 = Result = I can't connect to the server, because v1.1.3. Häääääääää

PLS HELP xDD

**Comment by cout970** [comment]
The latest version of Magneticraft ONLY requires modelloader in the CLIENT you don't need modelloader in the server anymore.

**Comment by TheSnowyChickens** [comment]
Ahhhh, I try it. ^^

**Comment by TheSnowyChickens** [comment]
Edit: without the modelloader, my server crash all the time. (at the start up)
(this crash is also in the linked issue from NillerMedDild)

[crash-2018-11-08_16.46.44-server.txt](https://github.com/Magneticraft-Team/Magneticraft/files/2562428/crash-2018-11-08_16.46.44-server.txt)

**Comment by cout970** [comment]
oh, I see the problem, a multiblock is using a function from the library in the server

**Comment by Pythonic-Rainbow** [comment]
Please reopen this, I am facing the exact same issue

**Comment by Pythonic-Rainbow** [comment]
https://github.com/Magneticraft-Team/Magneticraft/issues/152

**Comment by Pythonic-Rainbow** [comment]
@cout970

**Comment by cout970** [comment]
Just add the ModelLoader jar into the server

**Comment by Pythonic-Rainbow** [comment]
Doesn't work even with 1.1.5 @cout970

**Comment by cout970** [comment]
@Pythonic-Rainbow try with 1.1.6 https://minecraft.curseforge.com/projects/modelloader/files/2724525

**Comment by Pythonic-Rainbow** [comment]
The server says "Disabling ModelLoader as its client side only" (not a 1.1.6 new issue, appears since long time ago)
I'll still run the server with it, but why does this appear?

**Comment by Pythonic-Rainbow** [comment]
Fixed with 1.1.6, gonna close #152

**Comment by cout970** [comment]
Some versions ago, ModelLoader as loaded in the server and the client, but it's only purpose is to load models for the client, so I changed it to be client side only, then I got people complaining that Magneticraft wasn't loading on the server, that was because ModelLoaded adds a bunch of vector and matrix utilities and Magneticraft was using some functions on the server side. In this new release I added an annotation to make forge load only the utilities in the server, the rest of the library is only loaded in the client. 
That's why you get that the library is disable in the server.
